PluginFramework/Interfaces/PluginTypes/IPluginOverhead.cs

```diff
@@ -25,6 +25,6 @@ public interface IPluginOverhead
         /// This method will return the settings view for the plugin
         /// </summary>
         /// <returns>A useable control to implement into the host application</returns>
-        UserControl GetSettingsPage();
+        //UserControl GetSettingsPage();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5634024)